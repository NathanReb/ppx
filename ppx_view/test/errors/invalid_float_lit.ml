let f x =
  match%view x with
  | 1.0z -> ()
