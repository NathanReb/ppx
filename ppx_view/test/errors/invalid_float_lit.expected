File "invalid_float_lit.ml", line 3, characters 4-8:
Error: ppx_view: Unsupported float litteral suffix 'z'
